geometry_msgs/Point32[] vertices
geometry_msgs/Point32[] normals
geometry_msgs/Point32[] colors
int64[] labels